### ISSUE TITLE
Anti-adblock report: https://www.noticiasautomotivas.com.br/

### ISSUE THREAD

**Issue by jackdrits** [reek/anti-adblock-killer#3330]
https://www.noticiasautomotivas.com.br/
Mozilla Firefox
Greasemonkey
uBlock Origin
EasyList + AakList

**Comment by uBlock-user** [comment]
Anti Adblock script - https://noticiasautomotivas.com.br/cdn-cgi/apps/head/HNe_yV1PBZ1CWmgN9SPrJ_-W2ms.js

**Comment by uBlock-user** [comment]
Filter - `||noticiasautomotivas.com.br/cdn-cgi/*/*.js^$script`

**Comment by uBlock-user** [comment]
@jspenguin2017

**Comment by jackdrits** [comment]
Thank you!

**Comment by jackdrits** [comment]
Uh.. this filter also blocked DISQUS..
Is there a way not to block DISQUS?

**Comment by jspenguin2017** [comment]
Disqus? Where? Does Disqus load with uBO off?

**Comment by jackdrits** [comment]
That website has Disqus when you roll down the page.
Yep, works whehn uBO is off..

**Comment by jspenguin2017** [comment]
You got a test link?

**Comment by jspenguin2017** [comment]
Wait, they changed something.

**Comment by jspenguin2017** [comment]
This is going to be hard to implement for Userscript, I used background page to do a trick.

**Comment by jspenguin2017** [comment]
Inject this CSS from background page (and unblock the script before): `html, body { overflow:scroll; } cloudflare-app[app-id='no-adblock'] { display:none; }`

**Comment by uBlock-user** [comment]
Where's disqus being blocked ? Tell me what's changed. Something else must be breaking disqus.

**Comment by jspenguin2017** [comment]
I don't know, it seems to change every time I refresh, so I'm using CSS injection to fix it.

**Comment by uBlock-user** [comment]
This should kill the overlay for good `noticiasautomotivas.com.br##script:inject(overlay-buster.js)`

**Comment by jspenguin2017** [comment]
No need, I already patched it from extension, and I don't want to block all overlays, only that one.

**Comment by uBlock-user** [comment]
Nope, disqus is not making any connection whatsoever. I scrolled the entire page and opened few articles. Nothing

**Comment by jspenguin2017** [comment]
Remove your rule. 
![image](https://user-images.githubusercontent.com/7283682/27750814-b78dc9e6-5d96-11e7-8727-7c6e0f6f66c4.png)

**Comment by uBlock-user** [comment]
Which rule exactly ?

**Comment by jspenguin2017** [comment]
All custom rules for this domain, and update List and Extension.

**Comment by uBlock-user** [comment]
Okay, I can see disqus now. I was blocking scripts from CloudFlare as it also serves random anti adblock scripts on some domains I visit, so that was causing not to create a disqus connection. I can see it now.

**Comment by jackdrits** [comment]
hmmm
So you fixed it? I updated Aaklist + Reek from uBlock settings and removed the custom rule:

||noticiasautomotivas.com.br/cdn-cgi/*/*.js^$script

The anti-adblock is now detecting uBlock :(

**Comment by jspenguin2017** [comment]
Yea... I'm not Reek... 
Initially, I'm just here to collect the filter, but it turned out to be a bit more conversation... 
My new solution doesn't work on Firefox, so unfortunately you would need to wait for another contributor.

**Comment by jspenguin2017** [comment]
You can try the rule here: https://github.com/reek/anti-adblock-killer/issues/3330#issuecomment-312351314

**Comment by jackdrits** [comment]
That one didn't work..
Well, what i did was to put your rule back and when I wanna see Disquis i just disable uBlock lol
Thank you very much for your efforts!

**Comment by uBlock-user** [comment]
Nope, that filter works perfectly. You need to clear cache and site data otherwise the browser might be pulling from cache.

**Comment by jackdrits** [comment]
Cleared all cache and site data, still didn't work.
But thanks again mate!

**Comment by uBlock-user** [comment]
Are you on Firefox 54 ? That's the reason. WebExtensions don't work as expected. I'm on Chromium and it does the job.

**Comment by xR2** [comment]
I've added `noticiasautomotivas.com.br##script:inject(overlay-buster.js)` and works fine on my FF 54.0.1 (after cache clean up).
